public/note-app.html

```diff
@@ -54,7 +54,11 @@ <h5 class="card-title">Add a Note</h5>
         <hr>
         <h1>My Notes</h1>
         <hr>
-
+        <div class="row">
```

Comment:
This button should be placed in front of My Notes above the line.

![button](https://user-images.githubusercontent.com/59874304/109610069-429c3400-7b52-11eb-9d76-60235677ce70.JPG)